### ISSUE TITLE
'wind_force_avg_10min' reports value of 'wind_speed_avg_10min'

### ISSUE THREAD

**Issue by ruedi137** [metaodi/tecdottir#45]
Just noticed that the value of 'wind_force_avg_10min' reports the value of 'wind_speed_avg_10min'. Apparently the transformation from m/s to bft is not done

Example:
https://tecdottir.herokuapp.com/measurements/mythenquai?startDate=2023-08-07 reports
![Screenshot 2023-08-07 202547](https://github.com/metaodi/tecdottir/assets/99275921/16745063-0d28-4c50-aef3-0e44cfb45691)

**Comment by metaodi** [comment]
You're right! It seems this calculation is already missing in the source data. Let me check with the data provider. But it's rather easy to fix, maybe I'll quickly do it on my side.
